### ISSUE TITLE
Safari is not supported

### ISSUE THREAD

**Issue by urosran** [0x006F/react-media-recorder#19]
Hey there, 

Not an issue per se but safari is not supported, have you thought of using the polyfill to mend this? 

I am happy to help in any shape or form

**Comment by syeo66** [comment]
Might be harder than it seems. I was looking for a polyfill but it seems there is none (for video). Probably for a good reason.

**Comment by urosran** [comment]
Yeah, I forked your project to try to get at least the audio working but it's pretty hard. For some reason, it's not picking up onStopEvent. I'll put a PR so you can take a peak

**Comment by Jlomaka** [comment]
MediaRecorder API not supported on safari https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder

**Comment by 0x006F** [comment]
Yeah, the MediaRecorder isn't available on Safari. Correct me if I'm wrong, the polyfill you're using in #20 doesn't support video recording, right @urosran 
Like @syeo66 said, it might be for a good reason, that we aren't seeing any fully-fledged polyfill yet.

**Comment by urosran** [comment]
Yeah, it's audio only, dk if it would be possible to inject here to get at least full audio support

**Comment by 0x006F** [comment]
Okay, let me check. I haven't looked at your PR yet. Let's see what's going on there.

**Comment by startupgurukul** [comment]
One possible option is encode via web assembly 

https://github.com/GoogleChromeLabs/webm-wasm

**Comment by startupgurukul** [comment]
While continuing using mediarecorder polyfill for audio, and using above approach for video encoding, you may be one the few which who can record seamlessly on safari

 This deserves a Show HN as it can be useful for greater community..

**Comment by uvtech3012** [comment]
hi any update on safari as media recorder is not working on IOS devides?
